BGL/include/CGAL/boost/graph/parameters_interface.h

```diff
@@ -47,6 +47,7 @@ CGAL_add_named_parameter(geom_traits_t, geom_traits, geom_traits)
 CGAL_add_named_parameter(vertex_incident_patches_t, vertex_incident_patches, vertex_incident_patches_map)
 CGAL_add_named_parameter(density_control_factor_t, density_control_factor, density_control_factor)
 CGAL_add_named_parameter(use_delaunay_triangulation_t, use_delaunay_triangulation, use_delaunay_triangulation)
+CGAL_add_named_parameter(triangulate_visitor_t, triangulate_visitor, triangulate_visitor)
```

Comment:
Why not simply using `visitor`?

Reply:
There already was these two named parameters in the file: 
- `CGAL_add_named_parameter(graph_visitor_t, graph_visitor, **visitor**)`
- `CGAL_add_named_parameter(remeshing_visitor_t, remeshing_visitor, remeshing_visitor)`

so I decided to get with yet another visitor name...

We could also decide to merge them all in one NP `visitor`.

Reply:
Sure.